signac/contrib/project.py

```diff
@@ -1531,6 +1531,49 @@ def get_project(cls, root=None, search=True):
                 "Unable to determine project id for path '{}'.".format(os.path.abspath(root)))
         return cls(config=config)
 
+    @classmethod
+    def get_job(cls, root=None):
+        """Find a Job in or above the current working directory (or
+        provided path).
```

Comment:
The first paragraph in a doc-string should only span one line: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings